tech/languages/java/java-build-tools-installation.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Java Build Tools
+page: java
+---
+
+## Maven installation
+
+Maven is probably the most popular build tool in Java world these days. It's available in official Fedora repositories, in form of RPM. To install it, simply type:
+
+```
+# dnf install maven
+```
```

Comment:
I miss an example of installing a package from this build tools, also, are some of the libraries from Maven repositories packaged in Fedora?